### DISCUSSION TITLE
Welcome to RotationAndDeposition Discussions!

### DISCUSSION THREAD

**Discussion by MarchiyGV** [MarchiyGV/RotationAndDeposition#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dsamsonov1** [comment]
Надо бы описать воркфлоу с программой, а также назначение настроек в привязке к алгоритмам

Затравка описания: [описание_ver1.docx](https://github.com/MarchiyGV/RotationAndDeposition/files/9136078/_ver1.docx)

Также нужен верификационный пример.
Верификация: проверка математической корректности алгоритма. Валидация: проверка физической адекватности расчета.

Примеры, конечно, нужны обоих типов.

**Comment by dsamsonov1** [comment]
Насчет того, что при первом запуске идет компиляция кода - есть ли возможность об этом сказать пользователю? Ну и отключить возможность давить кнопки, пока она не закончится? Реально конфузит, когда ничего не происходит несколько минут.
